README.md

```diff
@@ -95,6 +95,11 @@ If you checked out the project from GitHub you can build the project with maven
 ```
 mvn clean install
 ```
+If you want to generate the jar from GitHub you run the following maven command:
+```
+mvn package
+```
+**NOTE** the jar file can be found in `javaparser/javaparser-core/target/javaparser-core-\<version\>-SNAPSHOT.jar
```

Comment:
Note that there are multiple jar files produced - one for "JavaParser Core", and one for "JavaParser Symbol Solver":

- `javaparser-core/target/javaparser-core-3.23.1-SNAPSHOT.jar`
- `javaparser-symbol-solver-core/target/javaparser-symbol-solver-core-3.23.1-SNAPSHOT.jar`